### PR TITLE
[feat]product-Listing-Page

### PR DESCRIPTION
**Pull request by abhij1607**
image

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -30,6 +30,7 @@
   --info-light-color: #cff4fc;
 
   --size-480: 48rem;
+  --size-320: 32rem
```
Comment (Hamdrive) [inline css/base.css]: forgot to put semicolon

File: product.html [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+      <nav class="brand-nav navigation box-shadow">
+        <div class="nav-col nv-col-1">
+          <div class="nav-brand">GLaDOS</div>
+
+          <ul class="list-non-bullets align-right">
+            <li class="list-item-inline">
+              <a class="link" href=""><i class="fas fa-heart"></i></a>
+            </li>
+            <li class="list-item-inline">
+              <a class="link" href=""
```
Comment (Hamdrive) [inline product.html]: Dont leave empty href, if you will be linking it lateron atleast fill it with # sign for the timebeing

File: product.html [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+        <div class="filter-price-container">
+          <h3>Price</h3>
+          <div class="filter-price-range flex-row">
+            <span>500</span>
+            <span>2500</span>
+            <span>5000</span>
+          </div>
+          <input
+            type="range"
+            name=""
```
Comment (Hamdrive) [inline product.html]: Give each input its name

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Great work done, good use of comments
Will suggest to refactor for better formatting, also look into typos in the code as mentioned

**Review by GuptaEesh — COMMENTED** [review]
Apart from the changes suggested by Hamza, everything looks good to merge.

File: css/base.css (hunk 1 above)
Comment (abhij1607) [reply css/base.css]: thanks for pointing, will add

File: product.html (hunk 2 above)
Comment (abhij1607) [reply product.html]: thank you for suggesting it, added # in all empty href

File: product.html (hunk 3 above)
Comment (abhij1607) [reply product.html]: Sure, thanks

**Review by Hamdrive — APPROVED** [review]
Looks good, changes suggested have been implemented
Ready to merge!

**Pull request merged**